packages/x-charts/src/BarChart/seriesConfig/tooltipPosition.ts

```diff
@@ -0,0 +1,48 @@
+import type { TooltipItemPositionGetter } from '../../internals/plugins/models/seriesConfig/tooltipItemPositionGetter.types';
+import { getBarDimensions } from '../useBarPlotData';
+
+const tooltipItemPositionGetter: TooltipItemPositionGetter<'bar'> = (params) => {
+  const { series, identifier, axesConfig, placement } = params;
+
+  if (!identifier || identifier.dataIndex === undefined) {
+    return null;
+  }
+  const itemSeries = series.bar?.series[identifier.seriesId];
+
+  if (series.bar == null || itemSeries == null) {
```

Comment:
If `series.bar` is null, the `itemSeries` will be null for sure, so we can simplify the condition.

```suggestion
  if (itemSeries == null) {
```

Reply:
TS is not that smart

It complains that `series.bar` could be undefined few lines after when using `series.bar.stackingGroups.length`